mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -764,13 +764,17 @@ class LinearRegressionSuite
           (Intercept) 6.3022157  0.0018600    3388   <2e-16 ***
           V2          4.6982442  0.0011805    3980   <2e-16 ***
           V3          7.1994344  0.0009044    7961   <2e-16 ***
+
+          # R code for r2adj
```

Comment:
This code doesn't work. 

```r
lm_fit <- lm(V1 ~ V2 + V3, data = d1)
summary(lm_fit)$r.squared
[1] 0.9998737
summary(lm_fit)$adj.r.squared
[1] 0.9998736
```

Reply:
I just want to confirm: is `summary(lm(X1 ~ ., data = part_00000))` this does not work on your side? Is it because you used different variable name or data name?

Reply:
What is `X1`? it's never defined.

Reply:
@tengpeng @sethah did you resolve this discussion?

Reply:
@srowen it's fine in terms of functioning.

Reply:
There may be some confusion. If you type that code, "as-is", into an R shell, it will not work. It reference a variable called `X1`, which is never defined. When we provide R code in comments like this, we intend for it to be copy and pasted into a shell and just work. So, it does not function.

Reply:
Thanks for the clarification. Do you think change `x1` to `V1` would help?

Reply:
Is there something wrong with the code I pasted above? That worked for me when I was using the R shell.

Reply:
Is the suggestion to use the code in https://github.com/apache/spark/pull/19638/files#r148852081 as the snippet here? that seems to show a different adjusted R2 than what the test asserts though. @tengpeng what's some R code that should execute given the code before this snippet that produces the R2 / adjusted R2 values below?

Reply:
1. Yes. 2. They(0.9999 and  0.9998736) are actually the same if on the same precision. 3. I only used import data. No other code.

Reply:
The confusion lies in my incorrect assumption that R Studio's naming convention when importing data is the same as R shell. R Studio uses `X_` for all variables including the dependent variable.
I have followed @sethah 's suggestion, replacing with `lm_fit <- lm(V1 ~ V2 + V3, data = d1)`. We should be fine here.